src/core/Akka.FSharp.Tests/Akka.FSharp.Tests.fsproj

```diff
@@ -28,7 +28,6 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="FSharp.Core" Version="7.0.200" />
```

Comment:
Was the issue here that FSharp.Core gets referenced automatically when we're using an F# project, therefore this is now duplicative?

Reply:
Got it.